enterprise/config/models.py

```diff
@@ -0,0 +1,41 @@
+"""
+Module for defining models needed for configuration of internal
+concerns like management command options and parameters.
+"""
+from config_models.models import ConfigurationModel
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+# pylint: disable=feature-toggle-needs-doc
+# This is not a feature toggle.
+class UpdateRoleAssignmentsWithCustomersConfig(ConfigurationModel):
+    """
+    Model that specifies parameters for the
+    ``update_role_assignments_with_customers`` management command.
+
+    .. no_pii:
+    """
+    class Meta:
+        app_label = 'enterprise'
+
+    role = models.TextField(
```

Comment:
nit: To prevent typos when writing out the name of a user role assignment, I'm wondering if this model field should provide a list of known choices/roles rather than an open text field?

While there is a check in the management command itself that this provided role is one of the expected valid roles, I feel this may cause some potential confusion if I run the management command / job with a typo'd role name within this `UpdateRoleAssignmentsWithCustomersConfig` configuration model.

Reply:
Oh, hmm, yeah that's a good idea.